cluster/kube/client_ingress.go

```diff
@@ -66,11 +67,20 @@ func kubeNginxIngressAnnotations(directive ctypes.ConnectHostnameToDeploymentDir
 		}
 	}
 
+	switch env["AKASH_PROVIDER_ISSUER_TYPE"] {
+	case "cluster-issuer":
+		result[fmt.Sprintf("%s/cluster-issuer", certManager)] = env["AKASH_PROVIDER_ISSUER_NAME"]
```

Comment:
1. it looks like we don't need both `AKASH_PROVIDER_ISSUER_TYPE` and `AKASH_PROVIDER_ISSUER_NAME` and call it `AKASH_PROVIDER_CERT_ISSUER`
2. `AKASH_PROVIDER_CERT_ISSUER` should be defined in the provider env section
3. `env["AKASH_PROVIDER_ISSUER_NAME"]` may return `""` if env is not set

Reply:
what are the values of both env variables in this case?

Reply:
In this case it would be `AKASH_PROVIDER_ISSUER_TYPE` = "cluster-issuer", `AKASH_PROVIDER_ISSUER_NAME` would be whatever the provider chose as the cert-manager ClusterIssuer name on the cluster. By default the name is `letsencrypt` as the default issuer is Let's Encrypt.

Reply:
By having `letsencrypt` value, the code in the provider can deduct, that issue is `cluster-issuer` ?

Reply:
it can't thus the `cluster-issuer` in the annotation, so the provider knows that its a ClusterIssuer called letsencrypt rather than a namespace scoped issuer called letsencrypt as well

Reply:
if workload asks the provider to issue a certificate wouldn't it always be the `cluster-issuer`?